ios/CollectorManagerBridge.m

```diff
@@ -11,5 +11,14 @@ @interface RCT_EXTERN_MODULE(CollectorManager, NSObject)
                   resolver:(RCTPromiseResolveBlock)resolve
                   rejecter:(RCTPromiseRejectBlock)reject
                   )
+                  
+RCT_EXTERN_METHOD(pinConfirm:(NSString *)name
+                  path:(NSString *)path
+                  method:(NSString *)method
+                  headers:(NSDictionary *)headers
+                  resolver:(RCTPromiseResolveBlock)resolve
+                  rejecter:(RCTPromiseRejectBlock)reject
+                  )
```

Comment:
This interface should change after revisiting the parameters that are used /needed here

Reply:
Done [here](https://github.com/Instawork/vgs-collect-react-native/pull/1/commits/65850574faec0cae00f0d2b793cf1529b5161f2f#diff-c10ccafa7325ad5fda7bcf295f9ec21845719860ce5973124b7ffa00e40b8bd5L16)